drivers/adc/ad717x/ad717x.c

```diff
@@ -483,7 +813,55 @@ int32_t AD717X_Init(ad717x_dev **device,
 	ret = AD717X_ReadRegister(dev, AD717X_ID_REG);
 	if(ret < 0)
 		return ret;
+	dev->active_device = init_param.active_device;
+	dev->num_channels = init_param.num_channels;
+
+	for (setup_index = 0; setup_index < ARRAY_SIZE(init_param.setups);
+	     setup_index++) {
+		/* Set Polarity */
+		ret = ad717x_set_polarity(dev, init_param.setups[setup_index].bi_unipolar,
+					  setup_index);
+		if (ret < 0)
+			return ret;
+
+		/* Select the reference source */
+		ret = ad717x_set_reference_source(dev,
+						  init_param.setups[setup_index].ref_source, setup_index);
+		if (ret < 0)
+			return ret;
+
+		/* Enable reference and input buffers */
+		ret = ad717x_enable_input_buffer(dev,
+						 init_param.setups[setup_index].input_buff,
+						 init_param.setups[setup_index].ref_buff,
+						 setup_index);
+		if (ret < 0)
+			return ret;
+	}
+
+	/* Set Conversion Mode */
+	ret = ad717x_set_adc_mode(dev, init_param.mode);
+	if (ret < 0)
+		return ret;
+
+	/*  Connect Analog Inputs, Assign Setup, Disable all channels */
+	for (channel_index = 0;
```

Comment:
nit: rename `channel_index` to something like `ch_index` so the `for` statement can fit one line. also remove extra spaces in the statement.

Reply:
Renamed variable to ch_index and accommodated the for loop in one single line. Removed extra spaces too